ports/stm32/boards/SPARKFUN_MICROMOD_STM32/mpconfigboard.h

```diff
@@ -0,0 +1,100 @@
+// The Sparkfun MicroMod spec uses a zero-based peripheral numbering scheme.
+// In cases where the 0th peripheral is the default, the "0" is omitted from
+// the name (e.g. "I2C" instead of "I2C0").
+//
+// Note: UART (UART0) is not present in the edge connector pinout because the
+// primary debug serial port is exposed as a virtual serial port over USB,
+// i.e. Serial.print() should print over USB VCP, not UART_TX1.
+//
+// For more details, see https://www.sparkfun.com/micromod#tech-specs
+
+#define MICROPY_HW_BOARD_NAME "SparkFun STM32 MicroMod Processor"
+#define MICROPY_HW_MCU_NAME "STM32F405RG"
+
+// 1 = use STM32 internal flash (1 MByte)
+// 0 = use onboard external SPI flash (16 MByte)
+#define MICROPY_HW_ENABLE_INTERNAL_FLASH_STORAGE (0)
+
+#define MICROPY_HW_HAS_FLASH (1)
+#define MICROPY_HW_ENABLE_RNG (1)
+#define MICROPY_HW_ENABLE_RTC (1)
+#define MICROPY_HW_ENABLE_DAC (1)
+#define MICROPY_HW_ENABLE_USB (1)
+
+// --- External SPI Flash config ---
```

Comment:
Please remove the `---` from all comments in this file, just to make it look consistent with other board config files.